hack/verify-style.sh

```diff
@@ -41,6 +45,9 @@ fi
 echo "Running go vet..."
 go vet $GOPKGS
 
+echo "Running dep check..."
+dep check
+
 # Temporariy disabled because it breaks openshift-ci
```

Comment:
we should fix this too while we're dealing with things...

Reply:
hah, it should just be removed, now that our types are in openshift/api